include/libserver/network/command/proto/RanchMessageDefinitions.hpp

```diff
@@ -552,9 +553,9 @@ struct RanchCommandSearchStallionOK
     uint32_t tid{};
     std::string name{};
     uint8_t grade{};
-    uint8_t chance{};
+    uint8_t inheritanceRate{};
     uint32_t matePrice{};
-    uint32_t unk7{};
+    uint32_t pregnancyChance{};  // Lower = higher success rate (0 = 64%, 30 = 2%)
```

Comment:
Use proper doxygen documentation and elaborate on this. What are the percentages?